### PR TITLE
Fix swagger

### PR DESCRIPTION
**Pull request by tupyy**
This PR fix errors and some warnings of the swagger file. 

Some of the warnings like `oas2-unused-definition` are not solved along with the `host missing warning`.

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -586,46 +594,20 @@ definitions:
...
-      name:
         type: string
+      name:
         description: Product name of the device (for example "UHD Graphics 620 (Whiskey Lake)")
-      address:
         type: string
+      address:
         description: Device address (for example "0000:00:02.0")
-
-  heartbeat:
```
Comment (masayag) [inline swagger.yaml:604]: the heartbeat is reported by the devices to the control plane.
it shouldn't be removed.

File: swagger.yaml [hunk 2]
```diff
@@ -3,13 +3,19 @@ info:
...
+    email: flotta@redhat.com
   license:
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0.html
 basePath: /api/flotta-management/v1
 tags:
   - name: devices
     description: Device management
+  - name: yggdrasil
+    description: Client daemon that establishes a receiving queue for instructions to be sent to the system via a broker
```
[the remark this answers is not in the document]
Comment (jakub-dzon) [reply swagger.yaml:18]: I think it's OK to mention yggdrasil here, as long as it is the only client communicating with the control plane.

Mentioning broker and messaging is misleading, though. We use HTTP for communication and not MQTT.

[the remark this answers is not in the document]
Comment (tupyy) [reply swagger.yaml:18]: in this context, I think it's more like grouping API together.

**Comment by tupyy** [comment]
@masayag What do you think about ignoring the rule `oas2-unused-definition`?

File: swagger.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (masayag) [reply swagger.yaml:18]: The tag is used to group the API calls for interacting with Yggdrasil daemon.
It defines the APIs that yggdrasil is using to communicate with the HTTP server.
so combining the two, can be: "API to communicate with Yggdrasil daemon over HTTP"
@jakub-dzon WDYT?

**Comment by masayag** [comment]
> @masayag What do you think about ignoring the rule `oas2-unused-definition`?

That's a good idea. I don't think we have any way to bypass it without ignoring it.

**Comment by masayag** [comment]
@tupyy build fails since there are missing code that should be generated based on the restored heartbeat object.
run `make all` locally and include in the PR the modified and models/heartbeat.go files).

Also, since there are 0 errors with this PR, let's include it as part of the `generate` rule in the Makefile (before calling `generate-from-swagger`) or if you think of a better place, (e.g. in `build` rule before `generate`).

This will allow the enforcement of the swagger validity.

**Review by masayag — APPROVED** [review]

File: Makefile [hunk 3]
```diff
@@ -198,7 +198,7 @@ ifeq (, $(shell which ginkgo))
 endif
 
 validate-swagger: ## Validate swagger
-	$(DOCKER) run -it --rm -v $(PWD)/.spectral.yaml:/tmp/.spectral.yaml:z -v $(PWD)/swagger.yaml:/tmp/swagger.yaml:z stoplight/spectral lint --ruleset "/tmp/.spectral.yaml" /tmp/swagger.yaml
+	$(DOCKER) run -t -it --rm -v $(PWD)/.spectral.yaml:/tmp/.spectral.yaml:z -v $(PWD)/swagger.yaml:/tmp/swagger.yaml:z stoplight/spectral lint --ruleset "/tmp/.spectral.yaml" /tmp/swagger.yaml
```
Comment (masayag) [inline Makefile:201]: can you remove `-t -it` and retest ?

Comment (masayag) [reply Makefile]: (added commit just to verify this one)

File: swagger.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jakub-dzon) [reply swagger.yaml]: Good idea.

**Review by jakub-dzon — APPROVED** [review]

**Pull request merged**